config/patternengines.json

```diff
@@ -1,5 +1 @@
-{
-    "patternengines": [
-        "\\PatternLab\\PatternEngine\\Twig\\PatternEngineRule"
-    ]
-}
+{"patternengines":["\\PatternLab\\PatternEngine\\Twig\\PatternEngineRule"]}
```

Comment:
Formatting-only change. Was this Intentional? If not, best to revert it so it doesn't pollute the diff.

Reply:
Reverted